macro/run_primary_vertexer_ITS.C

```diff
@@ -17,9 +17,11 @@
 #include "ITStracking/Vertexer.h"
 // #include "ITStrackingCUDA/VertexerTraitsGPU.h"
 
+#define GPUCA_O2_LIB
 #include "GPUO2Interface.h"
 #include "GPUReconstruction.h"
 #include "GPUChainITS.h"
+#undef GPUCA_O2_LIB
 #endif
```

Comment:
Are you sure you still need the #define and #under ?
I was hoping that I had that fixed. Can you let me know what is the problem without?

Reply:
idk, really, probably messed up something with rebasing. Indeed it's fixed.